health_check.md

```diff
@@ -0,0 +1,23 @@
+Health check endpoint guidelines
+================================
+
+When creating a new application a health check endpoint should be created as standard. A Lambda does not require a Health check at present.
+
+End point Url
+-------------
+This should be `/health` .
```

Comment:
Is  /healthcheck more usual?

Reply:
`/health` is the out of the box endpoint for Spring Boot Actuator so I left it at that, happy to change it if you have a strong opinion about it though Les.

Reply:
/health is fine by me.